lilio/traintest.py

```diff
@@ -57,26 +57,70 @@ def __init__(self, splitter: type[CVtype]) -> None:
 
     def split(
```

Comment:
You could add `@overload` here (from typing), to denote that the output type differs depending on the input type.

Reply:
Good idea! Now we use `@overload` instead of those type aliases.